packages/react/src/primitives/types/view.ts

```diff
@@ -2,83 +2,78 @@ import * as React from 'react';
 
 import { AriaProps, BaseComponentProps } from './base';
 import { BaseStyleProps } from './style';
+import { Root as SliderRoot } from '@radix-ui/react-slider';
+import {
+  Root as ExpanderRoot,
+  Item as ExpanderItem,
+} from '@radix-ui/react-accordion';
+import { Trigger as Tab } from '@radix-ui/react-tabs';
 
 type MergeProps<A, B> = A & Omit<B, keyof A>;
 
-export type ElementType = React.FC<any> | keyof JSX.IntrinsicElements;
+export type ElementType = React.ElementType;
 
-/**
- * @description
- * Convert string element type to DOMElement Type
- * e.g. 'button' => HTMLButtonElement
- */
-export type HTMLElementType<Element extends ElementType> =
```

Comment:
Removing this in favor of `React.ComponentRef`, it only supports class component type other than string element like `button` and functional component